examples/cord_ep/Makefile

```diff
@@ -25,7 +25,10 @@ USEMODULE += fmt
 DEVELHELP ?= 1
 
 # For debugging and demonstration purposes, we limit the lifetime to 60s
+# Set CONFIG_CORD_LT only if not being set via Kconfig
+ifndef CONFIG_CORD_LT
```

Comment:
This check only works after `include $(RIOTBASE)/Makefile.include`. That's where the Kconfig variables will be included (here and on `examples/cord_epsim/Makefile`).

Reply:
Updated :+1: